packages/askui-nodejs/src/utils/http/credentials.ts

```diff
@@ -5,14 +5,14 @@ export interface CredentialArgs {
 
 export class Credentials {
   constructor(
-    private credentials: CredentialArgs,
```

Comment:
The interface should be moved to an extra file. They don't belong together anymore. The `CredentialArgs` is now an interface for the public api of the `UiControlClient` so I would move it close to that.